src/objective-c/tests/End2EndTest/End2EndTest/AppDelegate.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AppDelegate.h
+//  End2EndTest
+//
+//  Created by mxyan on 7/11/16.
+//  Copyright © 2016 Google. All rights reserved.
+//
```

Comment:
Please use the same license headers as the other files. For what it's worth, you can just get rid of `AppDelegate.h` and `AppDelegate.m` if you want by cutting their content and pasting it at the beginning of `main.m`.